src/eei.cpp

```diff
@@ -177,8 +178,9 @@ inline int64_t maxCallGas(int64_t gas) {
       heraAssert(sp <= (1024 * stackItemSize), "EVM stack pointer out of bounds.");
       heraAssert(opcode >= 0x00 && opcode <= 0xff, "Invalid EVM instruction.");
 
-      auto it = evmInstructionNames.find(static_cast<uint8_t>(opcode));
-      string opName = (it != evmInstructionNames.end()) ? it->second : "UNKNOWN";
+      const char* opName = evmc_get_instruction_name_table()[static_cast<uint8_t>(opcode)];
```

Comment:
I'd like to have a check that this only gets `byzantium` opcodes since that is what evm2wasm "supports". The only way to do that with the current API is to build a local array while calling `get_instruction_metrics` every time. See my comments here: https://github.com/ethereum/evmc/pull/33